package.json

```diff
@@ -45,7 +45,10 @@
     "gulp-concat": "^2.6.0",
     "gulp-connect": "^2.0.6",
     "gulp-eslint": "^3.0.1",
+    "gulp-footer": "^1.0.5",
     "gulp-header": "^1.7.1",
+    "gulp-if": "^2.0.2",
+    "gulp-jscs": "^3.0.2",
```

Comment:
I added the `gulp-if` but am not sure what `gulp-jscs`s deal is.  If it was removed I might have accidentally re-added it after a merge conflict, I can remove it if its no longer needed.